specification/eventgrid/data-plane/Microsoft.Media/stable/2018-01-01/MediaServices.json

```diff
@@ -108,8 +108,837 @@
+              }
+            ],
+            "modelAsExtensible": true
+          },
+          "description": "Gets the Job output state."
+        }
+      },
+      "type": "object",
+      "required": [
+        "progress",
+        "state"
+      ],
+      "description": "The event data for a Job output."
+    },
+    "JobOutputAsset": {
+      "x-ms-discriminator-value": "#Microsoft.Media.JobOutputAsset",
+      "allOf": [
+        {
+          "$ref": "#/definitions/JobOutput"
+        }
+      ],
+      "properties": {
+        "assetName": {
+          "type": "string",
+          "description": "Gets the Job output asset name."
+        }
+      },
+      "type": "object",
+      "description": "The event data for a Job output asset."
+    },
+    "MediaJobOutputStateChangeEventData": {
+      "description": "Schema of the Data property of an EventGridEvent for a Microsoft.Media.JobOutputStateChange event.",
+      "type": "object",
+      "properties": {
+        "previousState": {
+          "type": "string",
+          "enum": [
+            "Canceled",
+            "Canceling",
+            "Error",
+            "Finished",
+            "Processing",
+            "Queued",
+            "Scheduled"
+          ],
+          "x-ms-enum": {
+            "name": "JobState",
+            "values": [
+              {
+                "value": "Canceled",
+                "description": "The job was canceled. This is a final state for the job."
+              },
+              {
+                "value": "Canceling",
+                "description": "The job is in the process of being canceled. This is a transient state for the job."
+              },
+              {
+                "value": "Error",
+                "description": "The job has encountered an error. This is a final state for the job."
+              },
+              {
+                "value": "Finished",
+                "description": "The job is finished. This is a final state for the job."
+              },
+              {
+                "value": "Processing",
+                "description": "The job is processing. This is a transient state for the job."
+              },
+              {
+                "value": "Queued",
+                "description": "The job is in a queued state, waiting for resources to become available. This is a transient state."
+              },
+              {
+                "value": "Scheduled",
+                "description": "The job is being scheduled to run on an available resource. This is a transient state, between queued and processing states."
+              }
+            ],
+            "modelAsExtensible": true
+          },
+          "description": "The previous state of the Job.",
+          "readOnly": true,
+          "x-nullable": false
+        },
+        "output": {
+          "$ref": "#/definitions/JobOutput",
+          "description": "Gets the output."
+        },
+        "jobCorrelationData": {
```

Comment:
Minor Nit: In MediaJobStateChangeEventData, a similar property is named "correlationData" while here it is "jobCorrelationData". Is this by design, or should these names be made consistent?

Reply:
This is intentional because this is in the JobOutputStateChange related messages.  The correlation data is a property of the Job object and not the JobOutput object (the Job contains the JobOutputs).

Reply:
Thanks, that sounds good.